notebooks/en/rag_llamaindex_librarian.ipynb

```diff
@@ -242,7 +242,7 @@
     "\n",
     "First let's start up the Ollama server. Unfortunately, there is no support in the [Ollama Python client](https://github.com/ollama/ollama-python) for actually starting and stopping the server itself, so we'll have to pop out of Python land for this.\n",
     "\n",
-    "In a separate terminal, run: `ollama serve`. Remember to terminate this after we're done here!"
+    "In a separate terminal, run: 'ollama pull llama2' and then`ollama serve`. Remember to terminate this after we're done here!"
```

Comment:
Good catch. However, I think you actually need to have `ollama serve` running **before** running `pull`.

```
> ollama pull zephyr
Error: could not connect to ollama app, is it running?
```